backend/account_v2/templates/login.html

```diff
@@ -104,7 +104,7 @@
     </style>
 </head>
 <body>
-    <form method="post" action="{% url 'login' %}">
+    <form method="post" action="{% url 'public:login' %}">
```

Comment:
@chandrasekharan-zipstack Why do we need to do this?

Reply:
@hari-kuriakose didn't mention it in detail in the PR - this was causing an issue in resolving the login API for OSS version
`NoReverseMatch("Reverse for 'login' not found. 'login' is not a valid view function or pattern name.")`

Reply:
@chandrasekharan-zipstack How was it working till now?

Reply:
In v1, it was a tenant app call and that got resolved